modules/ossm-security-auth-policy.adoc

```diff
@@ -0,0 +1,182 @@
+////
+Module included in the following assemblies:
+-service_mesh/v2x/ossm-security.adoc
+////
+
+[id="ossm-vs-istio_{context}"]
+= Configuring Role Based Access Control (RBAC)
+
+Role-based access control (RBAC) objects determine whether a user or service is allowed to perform a given action within a project. You can define mesh-, namespace-, and workload-wide access control for your workloads in the mesh. 
+
+To configure RBAC, create an `AuthorizationPolicy` resource in the namespace for which you are configuring access. If you are configuring mesh-wide access, use `istio-system`.
+
+For example, with RBAC, you can create policies that:
+
+* Configure intra-project communication.
+* Allow or deny full access to all workloads in the default namespace.
+* Configure ingress gateway access.
+* Require a token for access.
```

Comment:
I just realized that this isn't just a list of things you can do with AuthorizationPolicies, it's actually your mini-TOC.  I think it would make sense to use these bullet points as your headings below ("Require a token for access" sounds more like a something a user would search for than "restrict access with JSON Web Token.").

Reply:
I removed the steps from the last item. It really doesn't need to be a procedure if there are only two items. 

I don't agree with the final heading. I think that in the upstream, there's a specific topic about authenticating with JWT, not just any token. The security outline this work item references also mentions it. I think people will be looking for JWT specifically, and I think that the example is about JWT.